lib/classes/Utils.test.js

```diff
@@ -693,6 +694,7 @@ describe('Utils', () => {
         provider: {
           name: 'aws',
         },
+        package: {},
```

Comment:
Why is this change necessary?

Reply:
This was to fix a unit test bug as defined in #3600 that has since been merged. I'll revert and accept their changes.